driver/razermouse_driver.c

```diff
@@ -6062,6 +6099,7 @@ static void razer_mouse_disconnect(struct hid_device *hdev)
             break;
 
         case USB_DEVICE_ID_RAZER_BASILISK_V3:
+        case USB_DEVICE_ID_RAZER_BASILISK_V3_35K:
             device_remove_file(&hdev->dev, &dev_attr_poll_rate);
```

Comment:
you're missing the device_remove_file equivalents of the CREATE_DEVICE_FILE statements you added above